recipes/openal/all/conanfile.py

```diff
@@ -12,7 +12,7 @@ class OpenALConan(ConanFile):
     topics = ("openal", "audio", "api")
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://www.openal.org"
```

Comment:
```suggestion
    homepage = "https://openal-soft.org/"
```

And fix homepage as well.